nav-app/src/app/layer-upgrade/layer-upgrade.component.html

```diff
@@ -21,142 +32,147 @@ <h2>Layer Upgrade</h2>
         
-            <!-- select all objects in section -->
-            <div class="checkbox">
-                <input id="select_{{section}}" class="checkbox-custom" type="checkbox"
-                    [checked]="allReviewed(section)" (click)="reviewAllChanged(section)">
-                <label for="select_{{section}}" class="checkbox-custom-label noselect">select all</label>
+            <!-- review/un-review all objects in section -->
+            <div class="review-buttons">
+                <button mat-stroked-button (click)="reviewAll(section)" [disabled]="allReviewed(section)">Mark all as reviewed</button>
+                <button mat-stroked-button (click)="unreviewAll(section)" [disabled]="!anyReviewed(section)">Mark all as un-reviewed</button>
             </div>
 
-            <!-- techniques -->
-            <ng-container *ngFor="let attackID of changelog[section]">
-                <mat-expansion-panel *ngIf="!showAnnotatedOnly || anyAnnotated(attackID)">
-                    <mat-expansion-panel-header>
-                        <mat-panel-title>
-                            <div class="cols">
-                                <div class="float">
-                                    <mat-icon *ngIf="isReviewed(attackID)" class="reviewed">check_circle_outline</mat-icon>
-                                    <mat-icon *ngIf="!isReviewed(attackID)">radio_button_unchecked</mat-icon>
-                                </div>
-                                <div class="float"><b>{{attackID}}</b></div>
-                                <div class="float">{{getTechnique(attackID, viewModel).name}}</div>
-                            </div>
-                        </mat-panel-title>
-                    </mat-expansion-panel-header>
+            <mat-divider></mat-divider>
 
-                    <ng-template matExpansionPanelContent> <!-- technique lazy render -->
-
-                    <!-- version header -->
-                    <div class="cols">
-                        <div class="float wide">
-                            <div class="version">ATT&CK {{compareTo.version}}</div>
-                            <mat-divider></mat-divider>
-                        </div>
-                        <div class="float arrow"><mat-icon inline="true">east</mat-icon></div>
-                        <div class="float wide">
-                            <div class="version">ATT&CK {{viewModel.version}}</div>
-                            <mat-divider></mat-divider>
+            <!-- techniques -->
+            <mat-expansion-panel *ngFor="let attackID of applyFilters()[section]" #panel="matExpansionPanel">
+                <mat-expansion-panel-header>
+                    <mat-panel-title>
+                            <mat-icon *ngIf="isReviewed(attackID)" class="reviewed">check_circle_outline</mat-icon>
+                            <mat-icon *ngIf="!isReviewed(attackID)" class="disabled">radio_button_unchecked</mat-icon>
```

Comment:
The unchecked state here still appears as if it is clickable. It really needs a deemphasis color (lighter gray) to show that it's not supposed to be interacted with. See also: https://github.com/mitre-attack/attack-navigator/pull/318#discussion_r668050717